lib/charty/backends/pyplot.rb

```diff
@@ -2,109 +2,109 @@
 require 'fileutils'
 
 module Charty
-  class PyPlot < PlotterAdapter
-    Name = "pyplot"
-
-    def initialize
-      @plot = Matplotlib::Pyplot
-    end
+  module Backends
+    class Pyplot < Base
```

Comment:
I see, I know that it's officially Pyplot, not PyPlot 👍

Reply:
I needed to rename it to let `Backends.register_backend` generate `"pyplot"` from the class name.

Reply:
I got it.